src/helpers/cli.ts

```diff
@@ -185,6 +185,19 @@ const args: ICLIArgument[] = [
     type: 'string',
     description: 'Extra arguments to pass to gcov',
   },
+  {
+    alias: 'xc',
```

Comment:
Keep in mind that multicharacter short flags do not work with yargs.

Reply:
💯